spec/rails_graylog/logger_spec.rb

```diff
@@ -65,5 +65,19 @@
         subject.info(data_hash)
       end
     end
+
+    context 'when logging hash data with message and an additional field' do
+      let(:message_with_field) { { message: Faker::Lorem.sentence, other_field: 'custom value' }}
+
+      it 'calls the graylog notifier with the additional field' do
+        expect(notifier).to receive(:notify!).with(
+          message: message_with_field[:message], full_message: message_with_field[:message],
+          short_message: "#{message_with_field[:message].to_s[0...25]}...",
+          other_field: 'custom value',
```

Comment:
Does this make sense to normalize these hashes? So a bunch of meaningful keys will be in the root and the rest have to be in a different key called `extras`?